contracts/IbbtcVaultZap.sol

```diff
@@ -197,11 +207,37 @@ contract IbbtcVaultZap is PausableUpgradeable {
         return _vaultShares(crvLp);
     }
 
+    function expectedAmount(uint256[4] calldata _amounts, bool _mintIbbtc)
+        public
+        view
+        returns (uint256 amount)
+    {
+        uint256[4] memory depositAmounts = _constructDeposit(
+            _amounts,
+            _mintIbbtc
+        );
+
+        ERC20Upgradeable[4] memory assets = [
+            ERC20Upgradeable(address(IBBTC)),
+            ERC20Upgradeable(address(RENBTC)),
+            ERC20Upgradeable(address(WBTC)),
+            ERC20Upgradeable(address(SBTC))
+        ];
+        uint256 virtualPrice = CURVE_REN_POOL.get_virtual_price();
```

Comment:
Shouldn't this be Ibbtc pool? Also need to convert ibbtc to wibbtc

Reply:
done